packages/strapi-design-system/src/helpers/strings.ts

```diff
@@ -0,0 +1 @@
+export const stripReactIdOfColon = (str: string): string => str.replace(/:/g, '');
```

Comment:
This would only replace the first occurrence of a colon. If that is what we want, I'd suggest to use a different name for the helper. Also: I know it is very minor, but could please add a test?

Reply:
I suck at regex, how do I make it strip both out? I though /g would do that?

Reply:
`replaceAll(':', '')` would replace all of them, even if they would be somewhere in the middle of the string. If that is a concern you could do: `.replaceAll(/^:(.*):$/g, '$1')`.

Reply:
lets just do replace all